pkg/agent/multicluster/mc_route_controller.go

```diff
@@ -43,9 +51,16 @@ const (
 	minRetryDelay = 2 * time.Second
 	maxRetryDelay = 120 * time.Second
 
-	// Default number of workers processing a resource change
-	defaultWorkers = 1
-	workerItemKey  = "key"
+	workerItemKey = "key"
+
+	MulticlusterWireGuardPort = 51821
```

Comment:
Per our discussion, we can have a separate default port number for multi-cluster, but could we define the port number in the same file as apis.WireGuardListenPort?

Reply:
Done.